run_hls_fullpfalgo_mp7.tcl

```diff
@@ -2,7 +2,7 @@
 source config_hls_fullpfalgo_mp7.tcl
 
 # open the project, don't forget to reset
-open_project -reset "proj3-mp7-full"
+open_project -reset "proj3-mp7-full-ctp7"
```

Comment:
I think this change should be reverted (for the ctp7 you're running the `run_hls_fullpfalgo_ctp7.tcl`, not this file)